scripts/new_project.kts

```diff
@@ -10,37 +10,75 @@ object NewProject {
     private const val TEMPLATE_FOLDER_NAME = "CoroutineTemplate"
     private const val TEMPLATE_PACKAGE_NAME = "co.nimblehq.coroutine"
 
+    private const val PATTERN_APP = "^[A-Z\$][a-zA-Z0-9(\\s)\$]*\$"
+    private const val PATTERN_PACKAGE = "^[a-z]+(\\.[a-z][a-z0-9]*)*\$"
```

Comment:
@Tuubz It shorter than pattern in `newproject.sh`,  Please let me know if it doesn't work some case  🙏

Reply:
Aah, I see. 👍 

Unfortunately, the current reg-expattern allows package-name with only one package, but the minimum should be two

Ex: 
- `package-name=test` (should not be allowed)
- `package-name=test.example` (should be allowed)

What do you think? 🙏

Reply:
@Tuubz May be we need to update pattern a bit, Could you please try this `^[a-z]+(\.[a-z][a-z0-9]*)+$` for me  instead of `^[a-z]+(\\.[a-z][a-z0-9]*)*\$`🙏

Reply:
Nice, that works for me 🚀 

If I'm not mistaken, we could minimize it even a little bit more: `"^[a-z]+(\\.[a-z][a-z0-9])+$"`